.github/workflows/ci.yaml

```diff
@@ -34,26 +34,15 @@ jobs:
       with:
         python-version: ${{ env.PYTHON_VERSION }}
 
-    - name: Setup Environment
-      run: |
-        # librrd-dev: Needed for building rrdtool python bindings.
-        # libldap2-dev: Needed for building python-ldap.
-        # libsasl2-dev: Needed for building python-ldap.
-        # libkrb5-dev: Needed for building pykerberos.
-        sudo apt-get update
-        sudo apt-get install librrd-dev libldap2-dev libsasl2-dev libkrb5-dev
-        pip3 install pipenv
-    
-    - name: Cache virtual environment
-      id: cache-venv
```

Comment:
I'm all for deleting caching steps, which are not needed. But I am bit a surprised it is done in this commit.